python/lsst/ts/watcher/__init__.py

```diff
@@ -19,5 +19,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from .alarm import *
-from .version import *  # Generated by sconsUtils
+from .base import *
+from .model import *
+from .rules import *
+
+try:
+    from .version import *
+except ImportError:
+    pass
```

Comment:
Maybe issue a warning?

Reply:
I will if you think it important, but I would rather not otherwise because I'm not sure it tells the user much (except that they are running from source, which they presumably already know) and it requires importing the warnings module into the lsst.ts.watcher namespace (not a terrible thing to do, but still...).